config/env_blank.json

```diff
@@ -10,6 +12,7 @@
   "MAX_REQ_ATTEMPTS": 3,
   "WAREHOUSE_INCREMENT": 0,
   "CREATE_CSVS": false,
+  "PUBLISHED_DATE_WORKERS": 8,
```

Comment:
I increased up to 15 workers time take to run got decreased without errors(403). I might need to test what is the breaking point to start seeing 403's

Reply:
This is OK for now. We do not need spent more time to fine-tune the parameters further.